cmd/query/app/analytics/http_gateway.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package analytics
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+const (
+	routeGetDeepDependencies = "/api/deep-dependencies"
+)
+
+// HTTPGateway exposes analytics HTTP endpoints.
+type HTTPGateway struct {
```

Comment:
this isn't a "gateway". I think you just need a static function RegisterRoutes for now. see https://github.com/jaegertracing/jaeger/pull/6608/files